src/domUpdates.js

```diff
@@ -26,8 +38,8 @@ const domUpdates = {
 
   loadManagerPortal: (loginInfo, userData, roomData, bookingData) => {
     $('#customer-portal').hide();
-    let customers = mockUserData;
     hotel.sortHotelData(roomData, bookingData);
+    // let customers = userData;
```

Comment:
remove commented code before pushing to master

Reply:
Deleted